solutions/hierarchy/admin-sandbox/Kptfile

```diff
@@ -5,7 +5,7 @@ metadata:
   annotations:
     config.kubernetes.io/local-config: "true"
 info:
-  description: Creates an admin folder underneath testing
+  description: Creates an admin folder underneath testing.admins
```

Comment:
feels like you will need to merge from main

Reply:
Will do a rebase to get it caught up